xbmc/dialogs/GUIDialogMediaSource.cpp

```diff
@@ -487,7 +487,8 @@ void CGUIDialogMediaSource::UpdateButtons()
     std::string path;
     CURL url(item->GetPath());
     path = url.GetWithoutUserDetails();
-    if (path.empty()) path = "<" + g_localizeStrings.Get(231) + ">"; // <None>
+    if (path.empty())
+      path = g_localizeStrings.Get(1020); // "New path"
```

Comment:
@jjd-uk please adapt the comment according to changed string => // "Enter path..."